### PR TITLE
Backpack/ Text Fields

### PR DESCRIPTION
**Pull request by eadams17**
## WHAT
update TextFields in Backpack

## WHY
we are expanding our component library

## HOW
make tweaks to styling for `Input` component 

### Screenshots
(If applicable. Also, please censor any sensitive data)
Screen Shot 2024-06-11 at 5 11 42 PM

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)
https://www.notion.so/quill/Quill-Backpack-Buttons-Menus-Text-Fields-Segmented-Controls-Lists-85b0fa6c8cee4a108a015f307befb7a7?pvs=4

### What have you done to QA this feature?
(Provide enough detail that a reviewer could assess whether additional QA should be done. For larger projects, additionally use the Engineer Feature Testing Notion template. Review Guidelines if needed: https://www.notion.so/quill/Github-PR-QA-Guidelines-49e99fc965654ceeb8c6249bd9d181d7)
did a site-wide click through on staging to ensure that no weird formatting issues popped up and verified that inputs render and behave as expected

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes

### CODE REVIEW

File: services/QuillLMS/client/app/bundles/Shared/styles/input.scss [hunk 1]
```diff
@@ -114,9 +114,8 @@
   }
 
   label {
-    font-size: 12px;
-    font-weight: 600;
-    color: #009a80;
+    @include display-xs;
```
Comment (emilia-friedberg) [inline services/QuillLMS/client/app/bundles/Shared/styles/input.scss:121]: Yay!

File: services/QuillLMS/client/app/bundles/Shared/styles/input.scss [hunk 2]
```diff
@@ -339,10 +336,9 @@
       border-style: none none solid none;
     }
     label {
-      color: #7f7f7f;
-      font-size: 16px;
+      color: $quill-grey-90;
+      @include display-xs;
```
Comment (emilia-friedberg) [inline services/QuillLMS/client/app/bundles/Shared/styles/input.scss:343]: I think we might want to make a pattern of putting these mixins at the top of the style definition? What do you think?

File: services/QuillLMS/client/app/bundles/Shared/styles/input.scss [hunk 3]
```diff
@@ -56,10 +56,10 @@
...
     label {
-      color: #009a80 !important;
+      color: $quill-green !important;
     }
     .dropdown__control {
-      border: 2px #009a80 !important;
+      border: 1px $quill-green !important;
       border-style: none none solid !important;
     }
   }
```
Comment (emilia-friedberg) [inline services/QuillLMS/client/app/bundles/Shared/styles/input.scss]: Just confirming that we wanted to make this change to the dropdown here?

**Review by emilia-friedberg — APPROVED** [review]
Looks great! Two small comments. Very excited to see these mixins in action!

File: services/QuillLMS/client/app/bundles/Shared/styles/input.scss (hunk 2 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Shared/styles/input.scss:343]: I think that makes sense!

File: services/QuillLMS/client/app/bundles/Shared/styles/input.scss (hunk 3 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Shared/styles/input.scss]: ahh yeah, this should still be 2px-- good catch

**Pull request merged**